CHANGELOG.md

```diff
@@ -15,6 +15,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Added locidex for allele calling. See [PR 62](https://github.com/phac-nml/mikrokondo/pull/62)
 
+- Removed `--annotate` parameter from locidex search.
```

Comment:
I would just remove this from the changelog since it's already covered by the "Added locidex" statement above.

Reply:
will do, I was curiouis if it should be added. myself.

Reply:
As for if a difference is made, test data had different outputs. I will have to consult with James about the magnitude of differences. But it definately appears to be have altered the output data, the difference seems to come from how allelese are identified in the sequences. 

There is some logic I would have to get clarification on as running with extract first may resolve paralogs. 

The method which uses just `search` with the `--annotate` flag will identify alleles/openreading frames using pyrodigal which may have some down sides. But I beleive it is more efficient at a cost of accuracy

Running extract first creates a copy the fasta used to create the blast database into the working directory, then converts the input assembly into a blast database. The original database sequences are then blasted against the sample and some additional processing is done.

Reply:
but tl;dr we will have to seek further clarification on the impacts of each setting.